templates/files/k8s-resource/aws-cni.yaml

```diff
@@ -153,7 +153,7 @@ spec:
       tolerations:
         - operator: Exists
       initContainers:
-        - image: {{.DockerRegistry}}/giantswarm/aws-cni-init:v1.10.1-nftables
+        - image: {{.DockerRegistry}}/giantswarm/aws-cni-init:v1.11.0-nftables
```

Comment:
Do we want to go for 1.11.2? Adresses also a CVE

Improvement - https://github.com/aws/amazon-vpc-cni-k8s/pull/1991 
Improvement - https://github.com/aws/amazon-vpc-cni-k8s/pull/1996 
Improvement - https://github.com/aws/amazon-vpc-cni-k8s/pull/1997

Reply:
ok bumped

Reply:
Nope everything should be already done 💪🏻